sema/commons/j2/j2_functions.py

```
@@ -82,6 +82,36 @@ def xsd_format_date(content, quote, suffix):
     return xsd_value(asdt.isoformat(), quote, "xsd:date")
 
 
+def xsd_format_gyear(content, quote, suffix):
```

Ruff (0.8.2)

Missing return type annotation for public function `xsd_format_gyear`

(ANN201)